library/core/src/ptr/const_ptr.rs

```diff
@@ -1301,8 +1301,24 @@ impl<T: ?Sized> *const T {
     /// }
     /// # }
     /// ```
+    #[must_use]
+    #[stable(feature = "align_offset", since = "1.36.0")]
+    #[rustc_const_unstable(feature = "const_align_offset", issue = "90962")]
+    #[cfg(not(bootstrap))]
```

Comment:
instead of cfg-ing the entire function, you could do that only to the function body.